contracts/upgradeable/MultiCallerUpgradeable.sol

```diff
@@ -46,6 +52,25 @@ contract MultiCallerUpgradeable {
         //slither-disable-end calls-loop
     }
 
+    function tryMulticall(bytes[] calldata data) external returns (Result[] memory results) {
+        _validateMulticallData(data);
```

Comment:
Since we're modifying both `multicall` & `tryMulticall`, would it make sense to convert `_validateMulticallData` into a modifier of some sort?

Reply:
That's a good question. To my understanding, if we do end up writing some code in `_validateMulticallData` such as in Polygon_SpokePool.sol, using a modifier instead of the function would cost double the contract bytecode size for the included modifier logic. Apparently some of the SpokePool contracts are pretty close to the max size, and from testing, the gas difference between the two is pretty negligible.

Reply:
yeah modifiers cost more bytecode so i think its ok to keep as is